activetable.js

```diff
@@ -45,16 +50,27 @@ var ActiveTable = (function () {
                 if (typeof $input[0] === 'undefined') {
                     row_state.push($cell.text());
                 } else {
-                    data = $input.data()
+                    data = $input.data();
                     data.value = $('input', this)[0].value;
                     row_state.push(data);
                 }
             });
             state.push(row_state);
         }
+
+        function appendCol() {
+            column_widths.push(parseInt($(this).css('width')));
+        }
+
         $('#activeTable thead tr').each(appendRow);
         $('#activeTable tbody tr').each(appendRow);
-        return JSON.stringify(state);
+        $('#activeTable colgroup col').each(appendCol);
+        return JSON.stringify({
+            data: state,
```

Comment:
This is a result of the refactoring.  You are right, it is confusing, and I will rename everything.